bbb-common-web/src/main/java/org/bigbluebutton/api/Util.java

```diff
@@ -113,6 +116,23 @@ public static File getPresFileDownloadMarker(File presBaseDir, String presId, St
 		return null;
 	}
 
+	public static void deleteAlreadyMarkedDownloadableFiles(File presFileDir, String presId) {

```

Comment:
I was surprised we delete the actual file, meaning the case "enable A, enable B (thus disabling A), enable A" would cause an error since we'd have no file A at this point. However, @GuiLeme clarified we're deleting the _mark_ that the file is downloadable, not deleting the actual file. Let's rename this method to indicate this subtlety.

Reply:
What do you think about `clearDownloadableMarks` or `clearDownloadablePresentationMarks`?